src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2492,4 +2522,68 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         _playWarningBell->Run();
     }
+
+    Windows::UI::Xaml::DependencyProperty VisualBellLight::m_isTargetProperty =
+        Windows::UI::Xaml::DependencyProperty::RegisterAttached(
+            L"IsTarget",
+            winrt::xaml_typename<bool>(),
+            winrt::xaml_typename<winrt::Microsoft::Terminal::Control::VisualBellLight>(),
+            Windows::UI::Xaml::PropertyMetadata{ winrt::box_value(false), Windows::UI::Xaml::PropertyChangedCallback{ &VisualBellLight::OnIsTargetChanged } });

```

Comment:
Check out...
- https://github.com/microsoft/terminal/blob/main/src/cascadia/TerminalSettingsEditor/SettingContainer.h
   - there's a `DEPENDENCY_PROPERTY` macro I created there. You may be able to move this into `cppwinrt_utils` and use it yourself.
- https://github.com/microsoft/terminal/blob/0e8d5f24c4cf80e104cf6d16ae004c54015c13ce/src/cascadia/TerminalSettingsEditor/SettingContainer.cpp#L13-L64
   - We lazy load the dependency properties in `_InitializeProperties`. I'd take a look there and apply the same code pattern here.

Reply:
I think we'll probably need a different macro for this one since it involves getting/setting the property on a target (and I think we can wait on adding those until we actually want more lights in the code)